Marsiple/Common/Extensions/UIColor-Marsiple.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  UIColor.swift
+//  Marsiple
+//
+//  Created by Matej Korman on 18/07/2018.
+//  Copyright © 2018 Matej Korman. All rights reserved.
+//
+
+import UIKit
+
+extension UIColor {
```

Comment:
Very good, nice job! 👍 

P.s. How do YOU like this approach?

Reply:
It's better the the previous one. Very clean call site.